docs/articles/nunit/running-tests/NUnitLite-Runner.md

```diff
@@ -16,22 +16,35 @@ With NUnit 3.0, the NUnitLite framework has been merged with the full NUnit fram
 
 To run tests under NUnitLite, proceed as follows:
 
-* Create a console application to use as your test assembly. If you have already placed tests in a dll, you can change
-  the project so that it produces a console application and reload it.
+* You can use your existing unit test project (created with the `dotnet new nunit` command or from Visual Studio) or
+ create a new console application.  If you use an existing test project you will get a warning about duplicate entry points. This can be ignored.
+
+Using a standard test project has the benefit that you use both the standard `dotnet test` command, the Visual Studio
+ Test Explorer and the NUnitLite runner to run your tests.
 
 * Make sure your test assembly references both **nunit.framework** and **nunitlite**.
 
-* The Main() for your console application should look something like this...
+```xml
+<PackageReference Include="NUnit" Version="4.0.1" />
```

Comment:
ℹ️ Let me know if you're interested in turning this into a sample file so that we can keep the sample up to date with Dependabot and have the versions reflected here.

Reply:
I would like that.  I added a whole sample test project outside now.  Was unsure how to do this with the existing snippets, because this is more than a snippet, it is a complete test project created first with `dotnet new nunit`, thne updated to the latest package versions, and then added the nunitlite stuff.

It would also be nice to replace the screenshot further down in the article, it is from 2014 and with and old version.  But the test should then have more than just a single Assert.Pass.